tck/pom.xml

```diff
@@ -32,33 +32,6 @@
     <name>Jakarta Config TCK</name>
     <description>Jakarta Config :: TCK</description>
 
-    <dependencies>
```

Comment:
What's the point of deleting it?

Reply:
(In particular, see #116. Depending on its outcome, we would want to delete `jakarta.inject:jakarta.inject-api` for sure. I cannot comment on the others.)